cupy/cuda/device.pyx

```diff
@@ -247,6 +252,35 @@ cdef class Device:
         """
         return _get_attributes(self.id)
 
+    @property
+    def pci_bus_id(self):
+        """A string of the PCI Bus ID
+
+        Returns:
+            pci_bus_id (str):
+                Returned identifier string for the device in the following
+                format [domain]:[bus]:[device].[function] where domain, bus,
+                device, and function are all hexadecimal values.
+        """
+        return _get_pci_bus_id(self.id)
+
+    @classmethod
+    def from_pci_bus_id(cls, pci_bus_id):
```

Comment:
Let's move this definition to just after `__init__`, as this is a construction method.